.travis.yml

```diff
@@ -26,6 +26,8 @@ addons:
             - language-pack-de
             # tzdata is included to ensure system leap seconds are up to date.
             - tzdata
+            # The comparison tests with amuse need a fortran compiler
+            - gfortran
```

Comment:
This is a little scary...

Reply:
😱 for bringing so many dependencies into the CI picture

Reply:
Some compilers have problem installing in `conda` under OSX. I hope this is not one of them.

Reply:
To be honest, I don't think the base `amuse-frame` even needs a fortran compiler, but I wanted to check things worked before trying to work around this.

Reply:
> @rieder - you mentioned over at #10196 that you were worried about strings not sufficing - could you give explicit examples of amuse units that you think might be problematic?

Will do, Monday.